test/unit-tests/browserContext.test.js

```diff
@@ -1,162 +0,0 @@
-const expect = require('chai').expect;
```

Comment:
just want to confirm that this test can be deleted?

Reply:
yes as the deprecated api is now removed `openWindow` and `closeWindow` removed its corresponding test file too